skimage/transform/registration.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from scipy.optimize import basinhopping, minimize
+from scipy import ndimage as ndi
+
+from .pyramids import pyramid_gaussian
+from ..measure import compare_mse
+
+__all__ = ['register_affine']
+
+
+def _p_to_matrix(p):
+    """
+    Converts a transformation of the first 2 rows of a 3x3 transformation matrix
+    to the full matrix
+
+    Parameters
```

Comment:
Your docstring calls your inputs/outputs "param" and "matrix" but in the function code they are called "p" and "out".
1. Please replace these with more meaningful variable names.
2. Variable names should match in the docstring and the function code.